browser/components/molecules/PendingPrrrs/index.js

```diff
@@ -19,16 +19,6 @@ export default class PendingPrrrs extends Component {
 
   claimPrrr(prrr){
     claimPrrr(prrr.id)
-      .then(_ => {
```

Comment:
we still need to popup the PR. Lets have the server send back a message over the socket (only to us) when the Prrr is claimed. Out client side handler for that event. (maybe call if `prrrClaimedByYou`) should do this

```js
(prrr) => {
  const url = `https://github.com/${prrr.owner}/${prrr.repo}/pull/${prrr.number}`
  const popup = window.open(url, '_blank')
   if (popup) popup.focus()
}
```